src/elements/element.rectangle.js

```diff
@@ -50,42 +50,75 @@ module.exports = function(Chart) {
 	}
 
 	Chart.elements.Rectangle = Chart.Element.extend({
-		draw: function() {
+		draw: function(horizontal) {
```

Comment:
I still think that `horizontal` should be part of the model (`vm.horizontal`), not a draw argument.